docs/BRP-LO GBA mapping - Personen gba.csv

```diff
@@ -0,0 +1,137 @@
+7;67.20;code+omschrijving;G;;reden;;;;;
+7;68.10;ongewijzigd leveren (8 cijfers);;datumEersteInschrijvingGBA;;;;;;
+7;70.10;ongewijzigd leveren (0-7);G;geheimhoudingPersoonsgegevens;;;;;;
+8;09.10;code+omschrijving;G;gemeenteVanInschrijving;;;;;;
+8;09.20;ongewijzigd leveren (8 cijfers);;datumInschrijvingInGemeente;;;;;;
+8;;;;verblijfplaats;;;;;;
+8;10.10;code+omschrijving;G;;functieAdres;;;;;
+8;11.80;ongewijzigd leveren;;;adresseerbaarObjectIdentificatie;;;;;
+8;11.90;ongewijzigd leveren;;;nummeraanduidingIdentificatie;;;;;
+8;10.30;ongewijzigd leveren (8 cijfers);;;datumAanvangAdreshouding;;;;;
+8;13.20;ongewijzigd leveren (8 cijfers);;;datumAanvangAdresBuitenland;;;;;
+8;11.15;ongewijzigd leveren;;;naamOpenbareRuimte;;;;;
+8;11.10;ongewijzigd leveren;G;;straat;;;;;
+8;11.20;ongewijzigd leveren;G;;huisnummer;;;;;
+8;11.30;ongewijzigd leveren;G;;huisletter;;;;;
+8;11.40;ongewijzigd leveren;G;;huisnummertoevoeging;;;;;
+8;11.50;code+omschrijving;G;;aanduidingBijHuisnummer;;;;;
+8;11.60;ongewijzigd leveren;G;;postcode;;;;;
+8;11.70;ongewijzigd leveren;G;;woonplaats;;;;;
+8;12.10;ongewijzigd leveren;;;locatiebeschrijving;;;;;
+8;13.30;ongewijzigd leveren;G;;regel1;;;;;
+8;13.40;ongewijzigd leveren;G;;regel2;;;;;
+8;13.50;ongewijzigd leveren;G;;regel3;;;;;
+8;13.10;"code+omschrijving; ook 0000";G;;land;;;;;
+8;83;;;;inOnderzoek;;;;;
+8;83.10;ongewijzigd leveren (6 cijfers);;;;aanduidingGegevensInOnderzoek;;;;
+8;83.20;ongewijzigd leveren (8 cijfers);;;;datumIngangOnderzoek;;;;
+8;85.10;ongewijzigd leveren (8 cijfers);;;datumIngangGeldigheid;;;;;
+8;14;;;immigratie;;;;;;
+8;14.10;"code+omschrijving; ook 0000";;;landVanwaarIngeschreven;;;;;
+8;14.20;ongewijzigd leveren (8 cijfers);;;datumVestigingInNederland;;;;;
+9;;;;kinderen;;;;;;
+9;01.20;ongewijzigd leveren;;;burgerservicenummer;;;;;
+9;02;;;;naam;;;;;
+9;02.10;ongewijzigd leveren;;;;voornamen;;;;
+9;02.20;code+omschrijving+soort;;;;adellijkeTitelPredicaat;;;;
+9;02.30;ongewijzigd leveren;;;;voorvoegsel;;;;
+9;02.40;ongewijzigd leveren;;;;geslachtsnaam;;;;
+9;03;;;;geboorte;;;;;
+9;03.10;ongewijzigd leveren (8 cijfers);;;;datum;;;;
+9;03.20;code+omschrijving;;;;plaats;;;;
+9;03.30;code+omschrijving;;;;land;;;;
+9;83;;;;inOnderzoek;;;;;
+9;83.10;ongewijzigd leveren (6 cijfers);;;;aanduidingGegevensInOnderzoek;;;;
+9;83.20;ongewijzigd leveren (8 cijfers);;;;datumIngangOnderzoek;;;;
+10;;;;verblijfstitel;;;;;;
+10;39.10;code+omschrijving;;;aanduiding;;;;;
+10;39.20;ongewijzigd leveren (8 cijfers);;;datumEinde;;;;;
+10;39.30;ongewijzigd leveren (8 cijfers);;;datumIngang;;;;;
+10;83;;;;inOnderzoek;;;;;
+10;83.10;ongewijzigd leveren (6 cijfers);;;;aanduidingGegevensInOnderzoek;;;;
+10;83.20;ongewijzigd leveren (8 cijfers);;;;datumIngangOnderzoek;;;;
+11;32.10;code+omschrijving;;indicatieGezagMinderjarige;;;;;;
+11;33.10;als 1: true;;indicatieCurateleRegister;;;;;;Ik mis in 'gezagsverhouding.feature' nog een gba scenario waarin dit beschreven staat.
```

Comment:
dit staat in een kolom naast de issue kolom. Dit moet denk ik geen opmerking in deze sheet zijn, maar een GitHub issue, zodat we het op kunnen pakken.

Reply:
Ik heb een issue aangemaakt (zie #1201)